hera_cal/tests/test_frf.py

```diff
@@ -930,6 +931,97 @@ def test_load_dayenu_filter_and_write(self, tmpdir):
         os.remove(outfilename)
         shutil.rmtree(cdir)
 
+    @pytest.mark.parametrize("flip", [True, False])
+    def test_load_tophat_frfilter_and_write_with_filter_yaml(self, tmpdir, flip):
+        tmp_path = tmpdir.strpath
+        uvh5 = os.path.join(
+            DATA_PATH, "test_input/zen.2458101.46106.xx.HH.OCR_53x_54x_only.uvh5"
+        )
+        frate_centers = {(53, 54): 0.1}
+        frate_half_widths = {(53, 54): 0.05}
+        if flip:
+            filter_info = dict(
+                filter_centers={
+                    str(k[::-1]): -v for k, v in frate_centers.items()
+                },
+                filter_half_widths={
+                    str(k[::-1]): v for k, v in frate_half_widths.items()
+                },
+            )
+        else:
+            filter_info = dict(
+                filter_centers={str(k): v for k, v in frate_centers.items()},
+                filter_half_widths={str(k): v for k, v in frate_half_widths.items()},
+            )
+        frate_centers = {k+("ee",): v for k, v in frate_centers.items()}
+        frate_half_widths = {k+("ee",): v for k, v in frate_half_widths.items()}
+
+        with open(tmpdir / "filter_info.yaml", "w") as f:
+            yaml.dump(filter_info, f)
```

Comment:
One final suggestion: let's use the exmaple_filter_params.yaml in the test (that ensures it's not accidentally deleted in the future)

Reply:
Hm. How about I use the example filter file in the test for the missing baseline error message? The one that tests the actual performance of the code has two cases to make sure it does things correctly when the antennas are flipped in the filter parameter file, so I'm partial to using a temporary file for those tests.

Reply:
That's fine